driver/web/rest/apis.go

```diff
@@ -137,6 +142,10 @@ func (h *ApisHandler) CreateGroup(clientID string, current *model.User, w http.R
 		OnlyAdminsCanCreatePolls: requestData.OnlyAdminsCanCreatePolls,
 		CanJoinAutomatically:     requestData.CanJoinAutomatically,
 		AttendanceGroup:          requestData.AttendanceGroup,
+		ResearchGroup:            requestData.ResearchGroup,
+		ResearchOpen:             requestData.ResearchOpen,
+		ResearchDescription:      requestData.ResearchDescription,
+		ResearchProfile:          requestData.ResearchProfile,
```

Comment:
Hi @mdryankov, sorry one more comment I forgot to leave is that we want to restrict who can create a research group using the `research_group_admin` permission. This is very similar to what we ended up doing for Authman managed groups previously. I have assigned this permission to your account so you can make sure it works.

Reply:
Resolved